fe/fe-core/src/main/java/org/apache/doris/planner/ExportSink.java

```diff
@@ -44,6 +45,15 @@ public ExportSink(String exportPath, String columnSeparator,
         this.brokerDesc = brokerDesc;
     }
 
+    public ExportSink(String exportPath, String columnSeparator,
+                      String lineDelimiter, BrokerDesc brokerDesc, String header) {
+        this.exportPath = exportPath;
+        this.columnSeparator = columnSeparator;
+        this.lineDelimiter = lineDelimiter;
+        this.brokerDesc = brokerDesc;
```

Comment:
ExportSink(exportPath, columnSeparator, lineDelimiter, brokerDesc); may be better

Reply:
ok